README.md

```diff
@@ -141,6 +145,32 @@ faster executable.
 After compilation, the executable `scryer-prolog` is available in the
 directory&nbsp;`target/release` and can be invoked to run the system.
 
+### Via `cargo install`
+
+#### From git
+
+```
+cargo install --locked --git https://github.com/mthom/scryer-prolog.git
+```
+
+Afterwards the binary will be under `$HOME/.cargo/bin` which is usually added to your PATH 
```

Comment:
I believe it is "in" (not "under")?

A colleague once told me that, decades&nbsp;ago, they had to reinstall systems in a large organization and sent out instructions for employees to back up their personal&nbsp;files. The instructions stated that "all files *under* a certain directory" would be backed up, and one employee had their life's work destroyed because they took "under" to mean that the files have to be named to occur alphabetically *after* the named directory so that they would appear "under" the directory.

Reply:
I would have used "in" and "under" synonymous in this context.
I will change the wording anyways.